imports/ui/admin/Admin.js

```diff
@@ -0,0 +1,61 @@
+import './Admin.html';
+import { Meteor } from 'meteor/meteor';
+
+import Players from '../../api/Players';
+import Admin from '../../api/Admin';
+
+Template.Admin.events({
+    'change .token'(event) {
+        Session.set("admin", !!Admin.findOne({ token: event.target.value }));
```

Comment:
What happens if this database isn't fully set up/how is this database being set up>

Reply:
Ah, we have to manually set that up..